### PR TITLE
Part 3 responsive layout

### PR DESCRIPTION
**Pull request by araraltawarah98**

### CODE REVIEW

**Review by abood454 — APPROVED** [review]

**Review by ashrafabuodeh — APPROVED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

File: task_1/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (mmasaid) [inline task_1/.vscode/settings.json:3]: No need for the VS setting
```suggestion
```

File: task_1/index.html [hunk 2]
```diff
@@ -15,6 +15,9 @@
       <div class="logo">
         <img src="./assets/images/coffee-beans.png" alt="coffee beans">
       </div>
+      <div class="menu-icon">
+        <img src="./assets/images/menu-button-of-three-horizontal-lines.png" alt="menu icon">
```
Comment (mmasaid) [inline task_1/index.html:19]:
```suggestion
        <img src="./assets/images/menu-button-of-three-horizontal-lines.png" alt="Menu Icon">
```

File: task_1/index.js [hunk 3]
```diff
@@ -9,3 +9,7 @@ const onMenuItemClick = (elementId) => {
   }
   navItem.classList.add(active);
 };
+
+document.getElementsByClassName("menu-icon")[0].addEventListener("click", (e) => {
+  document.querySelector(".header .menu").classList.toggle('display-toggle');
```
Comment (mmasaid) [inline task_1/index.js:14]: 👍

File: task_1/style.css [hunk 4]
```diff
@@ -37,6 +37,7 @@ p {
 
 .header .menu {
   width: 50%;
+  min-width: 500px;
```
Comment (mmasaid) [inline task_1/style.css:40]:
```suggestion
   min-width: 500px;
   width: 50%;
```

File: task_1/style.css [hunk 5]
```diff
@@ -185,3 +196,87 @@ p {
...
+    width: 100%;
+  }
+
+  .header .display-toggle {
+    display: block;
+  }
+
+  .header .menu ul {
+    background: #eb5757;
+    display: block;
```
Comment (mmasaid) [inline task_1/style.css:268]: Use flex direction
```suggestion
....
```

File: task_1/style.css [hunk 6]
```diff
@@ -185,3 +196,87 @@ p {
...
+    margin-top: -15px;
+    position: relative;
+    z-index: 1;
```
Comment (mmasaid) [inline task_1/style.css:271]: No need for Z-index
```suggestion
```

**Review by mmasaid — CHANGES_REQUESTED** [review]
CR: address the comments
FR: Need to handle the flex-box

**Review by mmasaid — APPROVED** [review]

**Pull request merged**